src/i18n/config.ts

```diff
@@ -0,0 +1,17 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import en from './translations/en.json';
+
+i18n.use(initReactI18next).init({
+  resources: {
+    en: {
+      translation: en,
```

Comment:
Could we have `pl` as well? Each label we introduce should have translation in both `en` and `pl`. It will be easier to create translations right away instead of adding them later

Reply:
Could we also have modules? So each module has its own file and then we can do `useTranslation("myModule")`  to not use long translation keys?

Reply:
Languages added.
Modules added.